requirements/base.txt

```diff
@@ -53,7 +53,7 @@ django==2.2.17
     #   super-csv
 djangorestframework==3.12.2
     # via super-csv
-edx-celeryutils==0.5.7
+edx-celeryutils==1.0.0
```

Comment:
major version bump just due to dropping python 3.5 support, which shouldn't be a big issue for us here